tests/base.py

```diff
@@ -635,3 +659,7 @@ def _sigintHandler(*args):
 
 
 signal.signal(signal.SIGINT, _sigintHandler)
+# If we insist on test databases not existing when we start, make sure we
+# check right away.
+if 'newdb' in os.environ.get('EXTRADEBUG', '').split():
```

Comment:
I like re-using EXTRADEBUG personally, it alleviates the need to add more CMake code for each additional debugging feature.